packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -21,6 +21,8 @@
 
 
 class PortLink(BaseModel):
+    """ Input/Output is a reference to the output port of another node in the project """
+
```

Comment:
not too sure about that description. if the descriptions of the fields are not enough. what about:
```Represents a reference to another node port```

Reply:
cleaned up doc